bspump/pipeline.py

```diff
@@ -497,6 +498,17 @@ def _do_process(self, event, depth, context):
                         self.MetricsCounter.add("event.drop", 1)
                 return
 
+        if self.Sinks:
```

Comment:
I guess I'm confused how it worked before? Like there were single Sinks before that somehow got triggered right? Now we have the `Sinks` variable for multi sinks. What happens if the `Sinks` variable is set and there is also a single sink? I'm confused...

Reply:
There are multiple options, if the single sink consumes the event (as all proper sinks should), the function returns on line 499 and those multiple sinks will not be triggered. If it doesn't consume the event (faulty sink), the multiple sinks will process it. This behavior is not really wanted imo, but idea is to split the sink into the Sink variable even if it's a single one in the final version.